### DISCUSSION TITLE
Paginate

### DISCUSSION THREAD

**Discussion by Manojkumar-manu** [Shreyas-sonu/Menoj-s-Kitchen#3] [Q&A] [answered]
I wanted a Pagination for food category instead of grid is it possible

**Comment by Shreyas-sonu** [comment, marked as answer]
There are less data as of Now, Also to showcase the dishes it is better to display them together, if more data are there we can do it as a Carousel or Animation refer this https://github.com/Shreyas-sonu/30MinsChallenge-Gym-website.git
